website/docs/docs/cloud/connect-data-platform/connect-onehouse.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Connect Onehouse"
+id: connect-onehouse
+description: "Setup instructions for connecting Onehouse to dbt Cloud"
+sidebar_label: "Connect Onehouse"
+pagination_next: null
+---
+
+<Snippet path="dbt-onehouse" />
+
+<Constant name="cloud" /> Supports connecting to [Onehouse SQL](https://www.onehouse.ai/) using the Apache Spark Connector with the Thrift method.
+
+**Connect to a Onehouse SQL Cluster with the `dbt-spark` adapter.**
+
+## Requirements
+
+* For dbt Cloud, ensure your Onehouse SQL endpoint is accessible via external DNS/IP, whitelisting dbt Cloud IPs.
+* For dbt Core, you can SSH tunnel into the VPC to access the endpoint, without opening up ports.
```

Comment:
```suggestion
* For dbt Core, you can SSH tunnel into the VPC to access the endpoint, without opening up ports.
```

Please remove this section because this is a page for setting up on dbt cloud. I recommend having a new page for dbt core setup

Reply:
@andywalner FYI

Reply:
removed.